### ISSUE TITLE
How do i set the number of epochs, batch sizes, learning rate etc. when training yolov3?

### ISSUE THREAD

**Issue by rupeshcash** [pjreddie/darknet#1100]
I am complete newbie. I just want to train the Yolov3 on VOC  dataset for different epochs and batch sizes but i have got no clue where these parameters are in the code. :p

**Comment by jjwwczy** [comment]
cfg/yolov3-voc.cfg    
And don't forget to uncomment the training parameter before training.
